src/frontend/src/views/LandingPageV2/components/Footer/index.tsx

```diff
@@ -0,0 +1,87 @@
+import Image from "next/image";
+import React, { useEffect } from "react";
+import BiohubLogoImg from "src/common/images/cz-biohub-logo.png";
+import CZILogoImg from "src/common/images/czi-logo.png";
+import FooterLogo from "src/common/images/gen-epi-logo.svg";
+import { ROUTES } from "src/common/routes";
+import {
+  CZBiohubLogo,
+  CZContainer,
+  CZILogo,
+  CZLogoContainer,
+  FooterBottomContainer,
+  FooterBottomLink,
+  FooterBottomLinkDivider,
+  FooterBottomLinks,
+  FooterBottomSeparator,
+  FooterContainer,
+  FooterLogoContainer,
+  FooterPartnerships,
+  FooterTopContainer,
+  FooterTopLink,
+  FooterTopLinks,
+  Span,
+} from "./style";
+
+export default function Footer(): JSX.Element {
+  useEffect(() => {
```

Comment:
What does this script do? Can you add a comment here explaining what this is for?